### ISSUE TITLE
How can I run this code with my obj files?

### ISSUE THREAD

**Issue by okdalto** [gaolinorange/Automatic-Unpaired-Shape-Deformation-Transfer#10]
Hello, Dr. Gao, congratulations for your nice work.

I'm trying to run your code with my own obj files but I don't know where to start. 

Could you tell me the steps how to run the code?

Thank you!

**Comment by seokg** [comment]
Hi, thank you for the nice work.
I am currently trying to preprocess the data by using the script `get_feature17.m`, but I am having trouble understanding inputs of the script `densemesh` , `simmesh`, and `matpath`.
Also when I try to force the script, there seems to have a missing Matlab script `meshlp` which is used in `cotlp.m`
Could you explain some of the details?

**Comment by tommaoer** [comment]
@seokg Thanks for your interest in our work. For the script 'geo_feature17.m', for the three parameters you mentioned, you can set the same input for 'densemesh' and 'simmesh'. For the 'matpath', it is the path for saving the processed data. 
BTW, if you give different input for 'densemesh' and 'simmesh', it will calculate the correspondence for both inputs, the correspondence can be used to do the mesh pooling in the network. But the mesh pooling is an experimental feature, which is not included in the release code.
After, I checked the repository, the script 'meshlp' is located in the folder [matlabscript](https://github.com/gaolinorange/Automatic-Unpaired-Shape-Deformation-Transfer/blob/master/matlabscript/meshlp.mexw64). I think you must encounterrd some other problems, you should give me more details and let me help you to solve it.

**Comment by seokg** [comment]
Thank you for the fast reply!
I currently testing the code in Ubuntu 18.04 environment and Matlab R2020a.
I am not an expert on Matlab, but I am guessing `meshlp.mexw64` are only executable on Windows machine. 
Is there any workaround solution to run the Matlab code in Ubuntu Environment?
If not I will try on a Window machine later.

**Comment by tommaoer** [comment]
@seokg Yes, the script 'meshlp.mexw64' is a complied file on Windows System. Recently, we are working on the source code to transfer it to multiple OS for convenience, such as the recently released ACAP code. We will let you know as soon as we prepare it. We are very sorry about this. 
BTW, the script 'meshlp.mexw64' is on mesh information, such as the vertex positions, face indices, 1-ring neighbors for each vertex, cotangent weigh on each edge, and so on. I think these are very easy to be implemented or find some alternatives for some researchers familiar with this. 
So, if you encounter any problems, please feel free to contact us. :)
